include/nuSQuIDS/body.h

```diff
@@ -669,6 +669,9 @@ class EarthAtm: public Body{
     double x_ye_min;
     /// \brief Electron fraction at maximum radius.
     double x_ye_max;
+    
+    marray<double, 1> ESpace;
```

Comment:
I think `ESpace` should be removed everywhere. `EmittingEarthAtm::injected_neutrino_flux` should use `nuSQUIDS::GetERange()` on the passed `nusquids` object to find out what the node energies are at which it should evaluate the flux. This should simplify the code in a number of places (as all functions which exist solely to set/handle it can be removed as well).